packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -778,22 +780,21 @@ const ValidationEngine = {
   },
 
   registerValidatorInGroup(group, validator) {
-    const groupConfig = ValidationEngine.addGroup(group);
+    const groupConfig = ValidationEngine.addGroup(group, false);
     groupConfig.registerValidator.call(groupConfig, validator);
   },
 
-  _shouldRemoveGroup(group, validatorsInGroup) {
+  _shouldRemoveGroup(group, validators, isRemovable) {
     const isDefaultGroup = group === undefined;
-    const isValidationGroupInstance = group && group.NAME === 'dxValidationGroup';
-    return !isDefaultGroup && !isValidationGroupInstance && !validatorsInGroup.length;
+    return !isDefaultGroup && !validators.length && !isRemovable;
```

Comment:
[incorrect logic]
Now you return `true` here from `_shouldRemoveGroup` when it's `!isRemovable` (**not** removable). Vice versa,  we should be able to remove when it's removable! Please, reverse true/false value for this variable: it should be true when group is removable and false when it's not removable.